docs-src/descriptions/about-observables.jade

```diff
@@ -1,18 +1,18 @@
 h2#about-observables Intro to Streams and Properties
 
 p.
-  Kefir supports two types of observables — streams and properties.
-  Stream represents a stream of events. And property represents a value,
-  that changes over time. The value of a property changes in response of events,
+  Kefir supports two types of observable — streams and properties.
+  Streams represent sequences of events made available over time. And properties represent
+  values that change over time. The value of a property changes in response to events,
   which means that any stream may be easily converted to a property.
 
 figure
   img(src='docs-src/images/stream-and-property.png')
 
 p.
-  In practice, the only difference between two types of observables
-  is that property may have a current value. The process of subscribing
```

Comment:
The `may` here is intentional. When one creates property like this `stream.toProperty()` it has no current value until first value from the stream.